routes/auth.js

```diff
@@ -10,5 +10,7 @@ router.post('/login', asyncWrapper(authController.login))
 router.post('/logout', asyncWrapper(authController.logout))
 router.get('/activate/:link', asyncWrapper(authController.activate))
 router.get('/refresh', asyncWrapper(authController.refresh))
+router.get('/reset-password-link', asyncWrapper(authController.sendResetPasswordEmail))
```

Comment:
why get, not post? may be forgot-password endpoint, not reset-password-link?

Reply:
my mistake, will update soon